customization/dispvm-customization.md

```diff
@@ -57,6 +57,42 @@ It is possible to change the settings for each new Disposable VM (DispVM). This
 4.  Shutdown the qube (either by `poweroff` from qube's terminal, or `qvm-shutdown` from dom0 terminal).
 
 
+### Using static Disposable VMs for sys-* ###
+
+You can use a static DispVM for `sys-*` as long as it is stateless.
+For example, a `sys-net` using DHCP or `sys-usb` will work.
+`sys-firewall` will not work unless you have no custom rules set, because per VM rules are stored in a configuration file inside that AppVM.
+To create one:
+
+~~~
+qvm-create -C DispVM -l red <sys-VMName>
+qvm-prefs <sys-VMName> virt_mode hvm
+qvm-service <sys-VMName> meminfo-writer off
+qvm-pci attach --persistent <sys-VMName> dom0:<BDF>
```

Comment:
I think this require a little explanation - how to get those devices. For example by listing old VM. Also, usb controllers needs `-o no-strict-reset=True`.